harfbuzz-sys/Cargo.toml

```diff
@@ -15,8 +15,13 @@ links = "harfbuzz"
 build = "build.rs"
 
 [build-dependencies]
-pkg-config = "0.3"
-cmake = "0.1"
+pkg-config = { version = "0.3", optional = true }
+cmake = { version = "0.1", optional = true }
 
 [target.'cfg(any(target_os = "android", all(unix, not(target_os = "macos"))))'.dependencies]
-freetype = "0.4"
+freetype = { version = "0.4", default-features = false }
+
+[features]
+default = ["build-native-harfbuzz", "build-native-freetype"]
+build-native-harfbuzz = ["cmake", "pkg-config"]
+build-native-freetype = ["freetype/build-native-freetype"]
```

Comment:
Given my comments in the freetype PRs, I believe the changes to harfbuzz-sys can be removed.